sdk/eventhub/Azure.Messaging.EventHubs/stress/README.md

```diff
@@ -0,0 +1,63 @@
+# Event Hub Stress Tests
+This is a preliminary README. It demonstrates how to use the stress tests in the current state.
+
+In order to run the stress tests locally, the necessary resource connections need to be input through the command line interface. Test runs can call any of the following tests:
+- EventProducerTest : "EventProd"
+- EventBufferedProducerTest : "EventBuffProd"
+- BurstBufferedProducerTest : "BurstBuffProd"
+- ConcurrentBufferedProducerTest : "ConcurBuffProd"
+
+## Local Stress Test Runs
+### Setting up resources and packages
+```cmd
+(env) ~/stress/src> dotnet clean
+(env) ~/stress/src> dotnet build
+```
+
+### Running Tests
+When tests are run locally, Azure resources need to be created prior to running the test. This can be done through the Azure CLI, an ARM template or bicep file, or the Azure Portal. The user is required to input the connection strings upon request on the command line when the test is being run. For more information about what resources are needed for each test, see the "Scenario Information" section below. 
+
+The recommended approach is to run tests one at a time when running locally.
+To run any one test, run the following:
+```cmd
+(env) ~/stress/src> dotnet run -f netcoreapp3.1 BasicPublishReadTest local
+```
+
+## Deploy a stress test
+In order to deploy stress tests to be run in kubernetes clusters, run:
+```cmd 
+(env) ~/azure-sdk-for-net/eng/common/scripts/stress-testing/deploy-stress-tests.ps1 `
```

Comment:
There are a couple places where the `~` home directory is referenced in this readme, though not everyone will have the same path structure for their repo. I usually type examples like `<git root>/eng/common/...` though I imagine there's a better solution out there.